### ISSUE TITLE
Despite being on the same network, phone won't show up in browser.

### ISSUE THREAD

**Issue by stpnwf** [RobinLinus/snapdrop#404]
I am unable to transfer files on my Redmi Note 10 phone. I am running Lineage OS 19 (based on Android AOSP 12). This was also an issue when I was on Lineage OS 18.1 (Based on Android AOSP 11).  I am unsure if the fact it doesn't have Play Services is the cause for the problem.

When I open Snapdrop for the first time on the phone, it will detect browser (or on another phone) and the browser will detect the phone, but I am unable to send or receive files. 
After the app is reopened, nothing is detected anymore, both on phone and browser. However, if I clear Snapdrop's storage on the phone, the same behavior is observed: Shows up the first time app is opened, unable to send/receive any file, then if app is reopened it is not detected anymore. :/

Other apps like Sharik work fine. I am unsure what could be causing this issue.
I know it is not a browser issue, because Snapdrop works fine on other phones I have tested.
What could be causing this issue?

**Comment by Longshoez** [comment]
Yeah i think the service is currently down, i cant send files from pc to pc either

**Comment by stpnwf** [comment]
The issue I am experiencing has been going on for a while now. Unsure if it is the same as the one you are describing.

**Comment by xuJ14** [comment]
I have the same issue: can not find iphone(15.3) on my PC(win 10). I'm not sure if it is because of the school network. Can you please help me figure out.

**Comment by hello-smile6** [comment]
I've had this result **every time** I've tried to use Snapdrop with multiple devices during a period of almost a year, so I'd say this is definitely reproducible.
